intro-HPC/title.tex

```diff
@@ -5,7 +5,7 @@
 \vspace*{6\baselineskip}
 
 \Huge \strong{HPC Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 25 2020
```

Comment:
You shouldn't commit these dates, the deploy mechanism that creates the PDFs automagically replaces `DATEPLACEHOLDER`

Reply:
yeah I just saw it.